.eslintrc

```diff
@@ -0,0 +1,55 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "warnOnUnsupportedTypeScriptVersion": false,
+      "ecmaFeatures": {},
+      "ecmaVersion": 2018,
+      "sourceType": "module",
+      "project": "./tsconfig.json"
+    },
+    "settings": {},
+    "globals": {},
+    "plugins": ["@typescript-eslint", "jasmine"],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+    "env": {
+      "jasmine": true,
+      "browser": true,
+      "es6": true
+    },
+
+    "rules": {
+        "no-multiple-empty-lines": ["error"],
+        "no-unused-expressions": ["error"],
+        "no-trailing-spaces": "off",
+        "no-underscore-dangle": "off",
+        "no-new-wrappers": ["error"],
+        "guard-for-in": ["error"],
+        "no-console": ["error", { "allow": ["warn", "error"] }],
+        "no-bitwise": "off",
+        "camelcase": ["error"],
+        "no-caller": ["error"],
+        "eol-last": ["error", "always"],
+        "no-eval": ["error"],
+        "max-len": ["error", { "code": 140 }],
+        "quotes": ["error", "single", { "allowTemplateLiterals": true }],
+        "eqeqeq": ["error", "always", { "null": "ignore" }],
+        "curly": ["error", "all"],
+        "radix": ["error", "always"],
+
+        "@typescript-eslint/explicit-member-accessibility": ["error", { "accessibility": "no-public" }],
+        "@typescript-eslint/explicit-function-return-type": ["error", { "allowExpressions": true }],
+        "@typescript-eslint/no-parameter-properties": ["error", { "allows": ["private", "protected"] }],
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/interface-name-prefix": "off",
+        "@typescript-eslint/no-use-before-define": ["error", { "functions": false, "classes": false }],
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-unused-vars": ["error", { "args": "none" }],
+
+        "indent": "off",
+        "@typescript-eslint/indent": ["error", 4, { "FunctionExpression": { "parameters": "first" } }],
+
+        "semi": "off",
+        "@typescript-eslint/semi": ["error"]
+    }
```

Comment:
For the record, it looks like the following rules have changed.

No alternative:
- no-debugger
- no-empty
- one-line
- whitespace

Has alternative, but was not added here:
- class-name
- no-duplicate-variable
- no-eval
- no-switch-case-fall-through

See: [Roadmap](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md) for status of individual rules.

Reply:
The following rules exist in `@typescript-eslint/recommended` or in `eslint:recommended`
- class-name - [@typescript-eslint/recommended](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json#L13) 
- no-duplicate-variable - [eslint:recommended](https://github.com/eslint/eslint/blob/master/lib/rules/no-redeclare.js#L25)
- no-switch-case-fall-through - [eslint:recommended](https://github.com/eslint/eslint/blob/master/lib/rules/no-fallthrough.js#L63)
- no-debugger - [eslint:recommended](https://github.com/eslint/eslint/blob/master/lib/rules/no-debugger.js#L19)
- no-empty - [eslint:recommended](https://github.com/eslint/eslint/blob/master/lib/rules/no-empty.js#L24)

--- 
- no-eval - [.eslintrc](https://github.com/microsoft/SimpleRestClients/pull/54/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R43)

--- 
Only for `one-line`, `align` and `whitespace` there are no implementation, however, for instance for `whitespace` there is a recommendation to use Prettier if it is a critical rule we can add this tool.

Reply:
> Only for `one-line`, `align` and `whitespace` there is no implementation...

For the (updated) record, this is the list of what we are losing.

> @ms-markda `.eslintrc` extends `@typescript-eslint/recommended` and `eslint:recommended`...

Good stuff 👍 

> ...for `whitespace` there is a recommendation to use Prettier if it is a critical rule we can add this tool.

Prettier is very opinionated, and will bring in many more changes than just these pieces. So that is not really a fair comparison. [I don't like how Roadmap implied it is a substitute.] Prettier is its own discussion...